pkgcheck updates for February 2025 news (#979)
ropensci/roweb3: content/blog/2025-02-27-news-february-2025/index.Rmd

- [Tuesday March 4th 14:00 European Central (13:00 UTC)](https://ropensci.org/events/coworking-2025-03)

### New pkgcheck checks

> Our pkgcheck package used both for our submission checks, and in [pkgcheck-action](https://github.com/ropensci-review-tools/pkgcheck-action) now has two additional checks:

Reviewer: ```suggestion
Our [pkgcheck](https://docs.ropensci.org/pkgcheck/) package is used both for our [software review](/software-review/) submission checks, and in [pkgcheck-action](https://github.com/ropensci-review-tools/pkgcheck-action).
It now features two additional checks:
```

> - A check that packages [do not list excessive numbers of package _Imports_](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_num_imports).

Reviewer: ```suggestion
- A check that packages do not [list excessive numbers of package _Imports_](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_num_imports).
```

>   Packages which imports exceeding the 95% of current CRAN packages will receive an :eyes: note.

Reviewer: add how many that currently is?

Author: done, thanks

> - A check that repositories [do not name the default GitHub branch "master"](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_branch_is_master), and will fail if so.

Reviewer: ```suggestion
- A check that repositories do not [name the default GitHub branch "master"](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_branch_is_master), and will fail if so.
```

Reviewer: add a link to https://www.tidyverse.org/blog/2021/10/renaming-default-branch/? that itself links to some context

> The [pkgcheck-action](https://github.com/ropensci-review-tools/pkgcheck-action) has also been updated to run on Ubuntu 24.04, and to include quarto to enable packages to use quarto as a vignette engine.

Reviewer: ```suggestion
The [pkgcheck-action](https://github.com/ropensci-review-tools/pkgcheck-action) has also been updated to run on Ubuntu 24.04, and to include quarto to enable packages to use [Quarto](https://quarto.org/) as a vignette engine.
```

- [Tuesday March 4th 14:00 European Central (13:00 UTC)](https://ropensci.org/events/coworking-2025-03)

### New pkgcheck checks

Our [pkgcheck](https://docs.ropensci.org/pkgcheck/) package is used both for our [software review](/software-review/) submission checks, and in [pkgcheck-action](https://github.com/ropensci-review-tools/pkgcheck-action).
It now features two additional checks:

- A check that packages do not [list excessive numbers of package _Imports_](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_num_imports).
  Packages with numbers of imports exceeding the 95% of current CRAN packages (which is currently 21 imported packages) will receive an :eyes: note.
- A check that repositories do not [name the default GitHub branch "master"](https://docs.ropensci.org/pkgcheck/articles/list-checks.html#pkgchk_branch_is_master), and will fail if so.

>   See [this tidyverse blog post](https://www.tidyverse.org/blog/2021/10/renaming-default-branch/) for helpful advice on renaming default branches.

Author: @maelle The only way to add your suggested link was as this separate sentence. I'm happy to leave it, but also feel free to remove if you like.
